src/todos.js

```diff
@@ -0,0 +1,85 @@
+function displayTaks(id, todo, checked) {
+  const list = document.getElementById('list');
+  const done = checked ? 'done' : '';
+  const item = `
+  <li class="item ${done}">       
+              <input type="checkbox" id=${id} name="checkbox" ${checked}/>
+              <input type="text" id="Todo" value=${todo} name="todo" readonly/>
+            <i class="fa-solid fa-ellipsis-vertical" id=${id} name="update"></i>
+          </li>
+  `;
+  list.insertAdjacentHTML('beforeend', item);
+}
+
+export default class Todos {
+  constructor() {
+    this.TodoList = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  renderTodos() {
+    this.TodoList.sort((a, b) => a.index - b.index).forEach((todo) => {
+      const checked = todo.completed ? 'checked' : '';
+      displayTaks(todo.index, todo.description, checked);
+    });
+  }
+
+  addTodo(todo, id, status = false) {
+    const checked = status ? 'checked' : '';
+
+    displayTaks(id, todo, checked);
+
+    this.TodoList.push({
+      index: id,
+      description: todo,
+      completed: false,
+    });
+
+    localStorage.setItem('todos', JSON.stringify(this.TodoList));
+  }
+
+  completeTodo(element) {
+    if (element.checked) {
+      element.removeAttribute('checked');
+    } else {
+      element.setAttribute('checked', true);
+    }
+    element.parentNode.classList.toggle('done');
+    this.TodoList[element.id].completed = !this.TodoList[element.id].completed;
+    localStorage.setItem('todos', JSON.stringify(this.TodoList));
+  }
+
+  removeTodo(element) {
+    const cuurId = parseInt(element.id, 10);
+    element.parentNode.parentNode.removeChild(element.parentNode);
+    this.TodoList = this.TodoList.filter((todo) => todo.index !== cuurId);
+
+    // eslint-disable-next-line no-plusplus
+    for (let i = cuurId; i < this.TodoList.length; i++) {
```

Comment:
You have done great implementing the delete operation. However, currently, the delete operation does not seem to work. Kindly fix this.

Reply:
Thank you for bringing this issue to my attention. I will look into it